src/main/java/com/fasterxml/jackson/databind/util/LRUMap.java

```diff
@@ -1,7 +1,6 @@
 package com.fasterxml.jackson.databind.util;
 
-import java.io.*;
-import java.util.concurrent.ConcurrentHashMap;
+import com.fasterxml.jackson.databind.util.concurrentlinkedhashmap.ConcurrentLinkedHashMap;
```

Comment:
Ok, one minor change I'd request: let's shorten that `concurrentlinkedhashmap` into `clhm`. Also need `package-info.java` with a brief Javadoc to explain the package.

Reply:
You might start with this one and modify as you see fit, [package-info.java](https://github.com/ben-manes/concurrentlinkedhashmap/blob/master/src/main/java/com/googlecode/concurrentlinkedhashmap/package-info.java).